scripts/scr_add_man/scr_add_man.gml

```diff
@@ -1,47 +1,41 @@
-function scr_add_man(argument0, argument1, argument2, argument3, argument4, argument5, argument6, argument7, argument8, argument9, argument10) {
 
-	// argument0 = role
-	// argument1 = company
-	// argument2 = armourset
-	// argument3 = weaponset
-	// argument4 = gear
-	// argument5 = exp
-	// argument6 = name
-	// argument7 = chaos
-	// argument8 = gear provided?
-	// argument9 = home, shipXY, or default
-	// argument10 = mobility
+function scr_add_man(man_role, target_company, choice_armour, choice_weapons, choice_gear, spawn_exp, spawn_name, corruption, other_gear, home_spot, mobility_items) {
 
-	// That should be sufficient to add stuff in a highly modifiable fashion
+	//all of this will in time be irrelevant as calling new TTRPG_stats() and then calling the correct methods within the new itme will replace this but for
+	//now its easy enough to use this as the structs continue to be built.
+	// other_gear = gear provided?
+	// home_spot = home, shipXY, or default
 
+	// That should be sufficient to add stuff in a highly modifiable fashion
 
+	var non_marine_roles = ["Skitarii","Techpriest","Ranger","Crusader","Sister of Battle","Sister Hospitaler", "Ork Sniper", "Flash Git"]
```

Comment:
can we change these to string ids? e.g. "skitarii", "sister_of_battle"

Reply:
needs a separate commit as it's a complete refactor of ancillary unit roles to do this